style.css

```diff
@@ -86,3 +140,12 @@ input::placeholder {
   display: flex;
   justify-content: flex-end;
 }
+
+/* ---------------- */
+
+.copyright {
+  border: 0.2rem solid black;
+  padding-left: 2rem;
+  margin-top: 7rem;
+  font-weight: 900;
+}
```

Comment:
- [ ] [OPTIONAL]
Let's ensure that the footer (copyright) sticks to the bottom of all the pages, as seen in the wireframe design.

**Your design:**
![image](https://github.com/AdrianScharff/Awesome-Books/assets/12745474/beb8e189-ac31-40f9-be18-23cb13528281)

**Expected:**
![image](https://github.com/AdrianScharff/Awesome-Books/assets/12745474/853b327a-2706-466c-b9be-7b41e912aadb)